cg/services/orders/store_order_services/store_case_order.py

```diff
@@ -52,9 +45,7 @@ def _process_case_samples(self, order: OrderIn) -> dict:
         # submit new samples to lims
         new_samples = [sample for sample in order.samples if sample.internal_id is None]
         if new_samples:
-            project_data, lims_map = self.lims.process_lims(
-                lims_order=order, new_samples=new_samples
-            )
+            project_data, lims_map = self.lims.process_lims(order=order, new_samples=new_samples)
```

Comment:
fix signature with correct arguments

Reply:
Is that not for the work item that reworks the case order though?

Reply:
yes, I will resolve